### PR TITLE
 ATR-337: methods for AcuminatorSuppressionMerge tool

### PR DESCRIPTION
**Pull request by metallmonster**

### CODE REVIEW

File: src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs [hunk 1]
```diff
@@ -81,5 +81,33 @@ public override bool Equals(object obj)
...
-	}
+
+        public int CompareTo(SuppressMessage other)
+        {
+            if (Equals(other))
+            {
+                return 0;
+            }
+
+            var idComparison = string.Compare(Id, other.Id);
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs]: I think that comparison should be ordinal, case-sensitive.
`String.Compare` with two parameters uses the current culture to perform the comparison (see [the docs](https://docs.microsoft.com/en-us/dotnet/api/system.string.compare?view=netframework-4.7.2#System_String_Compare_System_String_System_String_)).

**Comment by dropsonic** [comment]
@metallmonster congrats on having #200 pull request! :)

File: src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs (hunk 1 above)
Comment (metallmonster) [reply src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs]: @dropsonic I agree

**Comment by metallmonster** [comment]
@dropsonic one more try

**Pull request merged**